### PR TITLE
Semjon json

### PR DESCRIPTION
**Pull request by Senbjorn**
without annotations

### CODE REVIEW

**Comment by Senbjorn** [comment]
with annotation

File: L5_reflection/src/main/java/ru/track/json/JsonWriter.java [hunk 1]
```diff
@@ -106,12 +124,20 @@ private static String toJsonMap(@NotNull Object object) {
...
-    private static String toJsonObject(@NotNull Object object) {
+    private static String toJsonObject(@NotNull Object object) throws IllegalAccessException {
         Class clazz = object.getClass();
-        // TODO: implement!
-
-
-        return null;
+        Field[] fields = clazz.getDeclaredFields();
+        Map<String, String> objfiels = new LinkedHashMap<>();
+        JsonNullable nullable = (JsonNullable) clazz.getAnnotation(JsonNullable.class);
```
Comment (tehnotrack) [inline L5_reflection/src/main/java/ru/track/json/JsonWriter.java]: Лучше сохранить boolean условие - boolean isNullable;

File: L5_reflection/src/main/java/ru/track/json/JsonWriter.java [hunk 2]
```diff
@@ -81,10 +88,21 @@ private static String toJsonCollection(@NotNull Object object) {
...
-    private static String toJsonMap(@NotNull Object object) {
-        // TODO: implement!
-
-        return null;
+    private static String toJsonMap(@NotNull Object object) throws IllegalAccessException {
+        StringBuilder jsonMap = new StringBuilder();
+        Map<Object, Object> map = (Map) object;
+        Map<String,String> convmap = new LinkedHashMap<>();
+        for (Map.Entry e: map.entrySet()) {
+            if (e.getKey().getClass().equals(String.class)
```
Comment (tehnotrack) [inline L5_reflection/src/main/java/ru/track/json/JsonWriter.java]: Я думаю можно унифицировать этот код для строк и не строк.

**Comment by Senbjorn** [comment]
It seems that the work is done. I don't think that it is a very good solution, but there is green on semaphore.

**Comment by tehnotrack** [comment]
принято

**Pull request closed without merging**